dev-vcs/git-remote-hg/git-remote-hg-1.0.0.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+PYTHON_COMPAT=(python2_7)
```

Comment:
This doesn't do anything without a proper eclass.

Reply:
OK. I'll remove that line. I assume I should put python:2.7 as a dep instead?